addons/crm/views/crm_lead_views.xml

```diff
@@ -1065,6 +1073,73 @@ if record:
             <field name="act_window_id" ref="crm_lead_action_pipeline"/>
         </record>
 
+        <record id="crm_lead_action_duplicates" model="ir.actions.act_window">
+            <field name="name">Similar Leads</field>
+            <field name="res_model">crm.lead</field>
+            <field name="view_mode">tree,form,kanban</field>
+            <field name="search_view_id" ref="crm.view_crm_case_leads_filter"/>
+        </record>
+
+        <record id="crm_lead_view_kanban_duplicates" model="ir.ui.view">
+            <field name="name">view.crm.duplicated.lead.kanban</field>
+            <field name="model">crm.lead</field>
+            <field name="mode">primary</field>
+            <field name="inherit_id" ref="crm.crm_case_kanban_view_leads"/>
+            <field name="arch" type="xml">
+                <xpath expr="//kanban" position="inside">
+                    <field name="id" invisible="1"/>
+                    <field name="probability" invisible="1"/>
+                </xpath>
+                <xpath expr="//div" position="attributes">
+                    <attribute name="t-att-style" add="
```

Comment:
Well did not see this one before, just spotting it. Why this inline custom styling ? We should avoid it at all costs. What is its purpose (same for herebelow) ?